### PR TITLE
xliff-filter parameter

### PR DESCRIPTION
**Pull request by pzotov**
Task https://trello.com/c/Gu2JiYqr

Usage:
`flowc1 xliff=1 xliff-filter=curator/,educator/,workflow/  learner/learner.flow`

### CODE REVIEW

File: tools/flowc/manipulation/xliff.flow [hunk 1]
```diff
@@ -21,12 +21,25 @@ prepareXLiffData(program : FiProgram) -> [string] {
...
-saveXliffData(program, filename) {
+saveXliffData(program : FiProgram, filename : string, xliffFilter : string) -> void {
 	if (program.config.verbose >= 1) {
 		fcPrintln("Extracting XLiff data...", program.config.threadId);
 	} 
 	setTranslationSimpleStringsApi();
-	data0 = prepareXLiffData(program);
+
+	program0 = if (xliffFilter != "") {
+		filterModulePrefixes = strSplit2WithoutLeave(xliffFilter, [":"]);
```
Comment (alstrup) [inline tools/flowc/manipulation/xliff.flow:31]: Normally we use , for seperation but maybe there is a good reason to use :?

**Review by alstrup — COMMENTED** [review]
Add to the help text as well so it is documented. In general, looks good.

File: tools/flowc/manipulation/xliff.flow (hunk 1 above)
Comment (pzotov) [reply tools/flowc/manipulation/xliff.flow:31]: No good reason, just looked at the separator in bash. I'll change it to ","

**Comment by pzotov** [comment]
Looks like I have incorrect java version so my compiled version of flowc can't work with our Jenkins. So I reverted my changes

**Pull request merged**